### PR TITLE
Updated latest version of the 0.16 spec

### PR DESCRIPTION
**Pull request by david-driscoll**

### CODE REVIEW

File: src/Protocol/Models/InitializeParams.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ public string RootPath
         /// The initial trace setting. If omitted trace is disabled ('off').
         /// </summary>
         [Optional]
-        public InitializeTrace Trace { get; set; } = InitializeTrace.Off;
+        public TraceValue TraceValue { get; set; } = TraceValue.Off;
```
Comment (mholo65) [inline src/Protocol/Models/InitializeParams.cs:60]: Will this rename break existing clients? Rename of property `Trace` to `TraceValue`.

Comment (david-driscoll) [reply src/Protocol/Models/InitializeParams.cs:60]: yes... that shouldn't be there, nice catch.  I renamed the enum to line up with the spec, but the property name should stay.

File: sample/SampleServer/Program.cs [hunk 2]
```diff
@@ -40,15 +44,18 @@ static async Task MainAsync(string[] args)
...
                         .SetMinimumLevel(LogLevel.Debug))
                     .WithHandler<TextDocumentHandler>()
                     .WithHandler<DidChangeWatchedFilesHandler>()
                     .WithHandler<FoldingRangeHandler>()
                     .WithHandler<MyWorkspaceSymbolsHandler>()
                     .WithHandler<MyDocumentSymbolHandler>()
-                    .WithHandler<SemanticTokens>()
+                    .WithHandler<SemanticTokensHandler>()
+                    // Linking here because the spec changed but vscode is behind
+                    .WithLink(TextDocumentNames.SemanticTokensFull, "textDocument/semanticTokens")
```
Comment (david-driscoll) [inline sample/SampleServer/Program.cs:57]: @TylerLeonhardt this lets us fake out the vscode client for now, until it's updated to line up with the spec.

**Comment by TylerLeonhardt** [comment]
yay!

**Pull request merged**